docs/sphinx_doc/user/inputs.rst

```diff
@@ -159,8 +159,7 @@ This section deals with parameters that control the simulation.
    **type:** Real number
 
    Initial timestep size (in seconds) used to initialize the simulation. 
-   Only activated if :input_param:`time.cfl` = true and parameter is positive, 
-   if this parameter is negative it is ignored.
+   Only activated if :input_param:`time.cfl` = true and if ``time.initial_dt`` is positive.
```

Comment:
Is `time.cfl` boolean? Isn't it a real number?

Reply:
dang, I don't know what I was thinking, I meant if fixed_dt < 0. I'll fix it. good catch